src/main/kotlin/com/openlattice/hazelcast/processors/shuttle/UpdateIntegrationEntryProcessor.kt

```diff
@@ -0,0 +1,59 @@
+package com.openlattice.hazelcast.processors.shuttle
+import com.dataloom.mappers.ObjectMappers
+import com.kryptnostic.rhizome.hazelcast.processors.AbstractRhizomeEntryProcessor
+import com.openlattice.shuttle.Flight
+import com.openlattice.shuttle.Integration
+import com.openlattice.shuttle.IntegrationUpdate
+import java.net.URL
+
+class UpdateIntegrationEntryProcessor(val update: IntegrationUpdate) :
+        AbstractRhizomeEntryProcessor<String, Integration, Integration>() {
+
+    companion object {
+        private val mapper = ObjectMappers.getYamlMapper()
+    }
+
+    override fun process(entry: MutableMap.MutableEntry<String, Integration>): Integration {
+        val integration = entry.value
+
+        update.environment.ifPresent { integration.environment = it }
+
+        update.s3bucket.ifPresent { integration.s3bucket = it }
+
+        update.contacts.ifPresent { integration.contacts = it }
+
+        if (update.maxConnections.isPresent) { integration.maxConnections = update.maxConnections }
+
+        if (update.callbackUrls.isPresent) { integration.callbackUrls = update.callbackUrls }
+
+        update.flightPlanParameters.ifPresent {
+            val flightPlanParameters = integration.flightPlanParameters
+            it.forEach { entry ->
+                val currentFlightPlanParameter = flightPlanParameters.getValue(entry.key)
+                val update = entry.value
+                if (update.sql.isPresent) { currentFlightPlanParameter.sql = update.sql.get() }
```

Comment:
I think the `ifPresent` pattern is a little cleaner, i.e.
```
update.sql.ifPresent { sql -> currentFlightPlanParameter.sql = sql }
```
and so on for the other optionals